build.gradle.kts

```diff
@@ -0,0 +1,199 @@
+import org.gradle.api.tasks.bundling.Jar
+import kotlin.concurrent.thread
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import java.io.InputStream
+import org.jetbrains.kotlin.gradle.dsl.Coroutines
+
+plugins {
+    `java-library`
+    `maven-publish`
+    kotlin("jvm") version "1.2.61"
+    id("net.saliman.properties") version "1.4.6"
+}
+
+fun gitBranch() =
+        if (System.getenv("TRAVIS_BRANCH") != null) System.getenv("TRAVIS_BRANCH")!!
+        else {
+            val r = "git rev-parse --abbrev-ref HEAD".execute(rootDir.absolutePath)
+            r.lineSequence().last()
+        }
+
+val branch = gitBranch()
+logger.info("On branch $branch")
+
+group = "be.bluexin"
+version = "${if (branch == "master") "" else "$branch-"}${prop("version_number")}.${prop("build_number")}"
+
+repositories {
+    jcenter()
+}
+
+configurations {
+    arrayOf(
+            "shade",
+            "shadeInPlace"
+    ).forEach {
+        create(it) {
+            get("compileOnly").extendsFrom(this)
+            get("testCompileOnly").extendsFrom(this)
+        }
+    }
+}
+
+dependencies {
+    api(kotlin("stdlib-jdk8"))
+    api("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    api("org.jetbrains.kotlinx:kotlinx-coroutines-core:0.19.3")
+    api("org.jetbrains.kotlinx:kotlinx-coroutines-jdk8:0.19.3")
+    api("org.jetbrains.kotlinx:kotlinx-coroutines-nio:0.19.3")
+
+    implementation("org.slf4j:slf4j-api:1.7.25")
+    implementation("io.github.microutils:kotlin-logging:1.6.10")
+    implementation("org.slf4j:slf4j-simple:1.7.25")
+
+    "shade"("net.java.dev.jna:jna:4.5.0")
+    "shadeInPlace"(files("libs"))
+    "shadeInPlace"(files("libsExt") {
+        builtBy("expandDiscordRPC")
+    })
+}
+
+tasks.withType<KotlinCompile> {
+    kotlinOptions {
+        jvmTarget = "1.8"
+        languageVersion = "1.2"
+        apiVersion = "1.2"
+    }
+}
+
+kotlin {
+    experimental.coroutines = Coroutines.ENABLE
+}
```

Comment:
you can state this in one line
`kotlin.experimental.coroutines = Coroutines.ENABLE`